lib/Cas/Protocol/SamlValidateResponder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SimpleSAML\Module\casserver\Cas\Protocol;
+
+use SimpleSAML\Configuration;
+use SimpleSAML\XML\Shib13\AuthnResponse;
```

Comment:
Not an issue on this PR, but be aware that Shib13/SAML1.1 support will be dropped from SSP in the near future